client/app.jsx

```diff
@@ -1,14 +1,54 @@
 import React from 'react';
 import Header from './components/header';
-import MapRoute from './pages/map-route';
+import LocationForm from './components/location-form';
+import Map from './components/map';
```

Comment:
There is already a `Map` data structure defined in the global scope. Rename this to `RouteMap`.